backend.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Could you add the motivation and explanation in the PR description to this file for future readers? Thanks!

Reply:
Sure, will do that in a minute.

Reply:
Thank you!